packages/tendermint-rpc/src/adaptors/v0-33/responses.ts

```diff
@@ -396,6 +397,18 @@ function decodeBroadcastTxCommit(data: RpcBroadcastTxCommitResponse): responses.
   };
 }
 
+function decodeValidatorSignature(signature: string): ValidatorSignature {
```

Comment:
Since you are just guessing at the algorithm anyway, let's just decode it and return it as Uint8Array.

https://github.com/confio/ts-relayer/blob/b0c348c417803fcdc4cac2bfe3b83413bf41a94d/src/codec/tendermint/types/types.ts#L189 is just a Uint8Array and I will need to convert into that to submit it anyway. The algorithm info is available in the validator pubkey (which is a different endpoint)

Reply:
Does this apply to the other responses too, eg voting, that were already using this?

Reply:
Voting? Which endpoint is that? consensus?

I would just run a local simapp and check the rpc response. Otherwise, I suggest looking at tendermint/types dir

Reply:
`ValidatorSignature` is also used in the `Vote` type, but looks like this is just exported. I'll update to remove the algorithm guess.